### ISSUE TITLE
half HASSes

### ISSUE THREAD

**Issue by npfoss** [sipb/courseroad2#249]
Many 6-unit music classes can be combined to fill one full HASS-A requirement. Georgia and I think the best solution is to add the HASS-A attribute to those subjects and also add a flag for being a half-class. Then we can add them up in the audit properly and do the needful on the frontend.

How do you feel about this @venkatesh-sivaraman?

**Comment by venkatesh-sivaraman** [comment]
I agree that this is a good thing to add; however, I think it will have to be manual because I can’t see anything in the course catalog that differentiates these courses (short of just assuming all music classes are HASS-As of some variety). So it might need to wait on the catalog parser improvements, which I’m still working on and will allow attribute overrides for specific courses.

**Comment by georgiashay** [comment]
I took a look at the course catalog and it seems like the 21M.1xx and 21M.4xx classes are all half-hasses, but we might want to check with the department first before assuming this?

**Comment by venkatesh-sivaraman** [comment]
Half HASSes can now be marked as such in the catalog with an attribute called `is_half_class`. I added these manually using the new corrections UI (go to [fireroad.mit.edu](fireroad.mit.edu), log in and then go to Admin > Catalog Corrections). The catalog update will take effect tomorrow.
